### ISSUE TITLE
Is the Mapillary the research edition or commercial edition?

### ISSUE THREAD

**Issue by macn3388** [openseg-group/openseg.pytorch#6]
As the title.
And any config details or suggestions about pretraining on Mapillary will be appreciated!

Thanks!

**Comment by PkuRainBow** [comment]
Research edition. 

We choose crop size as 1024x1024, batch size as 16, max-iterations as 500000 and initial learning rate 0.01~ We believe larger crop size and longer training iterations could further improve the performance considering the Panoptic-DeepLab reports much higher performance on Mapillary validation set (57% _vs_ Ours 50.8%)
